docs/body.md

```diff
@@ -598,6 +598,26 @@ when this happens.
     or "10.1.1.1" to all be definitely identified as "my.example.com", without
     the ambiguity that could exist if zOpenStackHostMapSame were used.
 
+* zOpenStackHostLocalDomain
```

Comment:
I've already got the body.md open to update the docs, so I added this property to the Configuration Properties list.  Can check that off your list.